Sources/Workspace/InitPackage.swift

```diff
@@ -488,20 +508,96 @@ public final class InitPackage {
         }
     }
 
-    private func writeTestFileStubs(testsPath: AbsolutePath) throws {
-        let testModule = testsPath.appending(RelativePath(pkgname + Target.testModuleNameSuffix))
+    private func writeTestFileStubs(template: PackageTemplate, testsPath: AbsolutePath) throws {
+        let testModule = testsPath.appending(RelativePath(packageName + Target.testModuleNameSuffix))
         progressReporter?("Creating \(testModule.relative(to: destinationPath))/")
         try makeDirectories(testModule)
 
         let testClassFile = testModule.appending(RelativePath("\(moduleName)Tests.swift"))
-        switch packageTemplate.packageType {
+        switch template.packageType {
         case .systemModule, .empty, .manifest, .`extension`: break
         case .library:
             try writeLibraryTestsFile(testClassFile)
         case .executable:
             try writeExecutableTestsFile(testClassFile)
         }
     }
+
+    private func getSwiftPMDefaultTemplate(packageType: InitPackage.PackageType,
```

Comment:
iirtc, we only use this in one place now and its always in "new" mode, so maybe we can just remove the function and construct the default template struct there.